custom_components/recycle_app/api.py

```
@@ -163,9 +239,18 @@
 
 
 class FostPlusApiException(Exception):
+    """Base class for all FostPlus API related exceptions."""
+
     def __init__(self, code: str) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)